src/app.ts

```diff
@@ -26,4 +27,5 @@ app.get('/index', async (req: express.Request, res: express.Response) => {
     res.render('index.html');
 })
 
-app.get('/job-roles', getOpenJobRoles);
\ No newline at end of file
+app.get('/job-roles', getOpenJobRoles);
+app.get('/job-roles/:id', getJobRoleById);
```

Comment:
this should be the controller method -> not the service one

Reply:
oh so that's why it wasn't working LOL thank you!